inductiva/structures/simulators/fenicsx.py

```diff
@@ -20,21 +20,25 @@ def api_method_name(self) -> str:
 
     def run(
         self,
-        input_dir: types.Path,
-        commands: List[dict],
+        mesh_path: types.Path,
+        bc_path: types.Path,
+        material_path: types.Path,
```

Comment:
Make these refer to the filenames and not paths. The idea is that these files will live inside the same directory, `input_dir`.

Unless you think it's useful to load them from different directories? Is that the workflow you expect?

Reply:
I was considering starting with the material, mesh, and bc datasets and then running the simulations. However, if it makes things easier, we can opt to copy the required files instead

Reply:
what do you think?